tools/pd-ctl/README.md

```diff
@@ -189,6 +190,12 @@ This option only works when key type is "table".
     >> config set enable-cross-table-merge true  // Enable cross table merge.
     ```
 
+- `key-type` is key type. There are some strategics supported: ["table", "raw", "txn"], default: "table". When key type is "raw" or "txn", PD will be allowed to merge region cross table. 
```

Comment:
`key-type is key type` looks verbose... how about `key-type specifies the key encoding type used by the cluster.`